src/github.com/getlantern/flashlight/logging/logging.go

```diff
@@ -90,10 +93,14 @@ func Configure(addr string, cloudConfigCA string, instanceId string,
 
 	// Using a goroutine because we'll be using waitforserver and at this time
 	// the proxy is not yet ready.
+	done = make(chan bool, 1)
```

Comment:
Previously this code didn't block. I did this so I could bock this call without affecting other code.